### PR TITLE
fix(Pagination): handle case when titles value is not a direct object expression

### PR DESCRIPTION
**Pull request by adamviktora**
Closes https://github.com/patternfly/pf-codemods/issues/508

Should cover most cases, where `titles` value is not a direct object expression.

This will be fixed correctly:

```
import * as React from "react";
import { Pagination } from "@patternfly/react-core";

let titlesObject = {
  currPage: "test",
  paginationTitle: "test",
  toFirstPage: "test",
  toLastPage: "test",
  toNextPage: "test",
  toPreviousPage: "test",
  optionsToggle: "test",
};

titlesObject.paginationTitle = "test";
titlesObject["paginationTitle"] = "test";
titlesObject = {
  currPage: "test",
  paginationTitle: "test",
  toFirstPage: "test",
  toLastPage: "test",
  toNextPage: "test",
  toPreviousPage: "test",
  optionsToggle: "test",
};

<Pagination titles={titlesObject} />;

<Pagination
  titles={{
    currPage: "test",
    paginationTitle: "test",
    toFirstPage: "test",
    toLastPage: "test",
    toNextPage: "test",
    toPreviousPage: "test",
    optionsToggle: "test",
  }}
/>;

const paginationTitle = "test";
const titlesObject2 = {paginationTitle};

<Pagination titles={titlesObject2} />;
```

to this:

```
import * as React from "react";
import { Pagination } from "@patternfly/react-core";

let titlesObject = {
  currPageAriaLabel: "test",
  paginationAriaLabel: "test",
  toFirstPageAriaLabel: "test",
  toLastPageAriaLabel: "test",
  toNextPageAriaLabel: "test",
  toPreviousPageAriaLabel: "test",
  optionsToggleAriaLabel: "test",
};

titlesObject.paginationAriaLabel = "test";
titlesObject["paginationAriaLabel"] = "test";
titlesObject = {
  currPageAriaLabel: "test",
  paginationAriaLabel: "test",
  toFirstPageAriaLabel: "test",
  toLastPageAriaLabel: "test",
  toNextPageAriaLabel: "test",
  toPreviousPageAriaLabel: "test",
  optionsToggleAriaLabel: "test",
};

<Pagination titles={titlesObject} />;

<Pagination
  titles={{
    currPageAriaLabel: "test",
    paginationAriaLabel: "test",
    toFirstPageAriaLabel: "test",
    toLastPageAriaLabel: "test",
    toNextPageAriaLabel: "test",
    toPreviousPageAriaLabel: "test",
    optionsToggleAriaLabel: "test",
  }}
/>;

const paginationAriaLabel = "test";
const titlesObject2 = {paginationAriaLabel};

<Pagination titles={titlesObject2} />;
```

### CODE REVIEW

File: packages/eslint-plugin-pf-codemods/test/rules/v5/pagination-rename-props.js [hunk 1]
```diff
@@ -114,5 +110,65 @@ ruleTester.run("pagination-rename-props", rule, {
         },
       ],
     })),
+    ...Object.keys(updatedTitlesPropNames).map((oldName) => ({
+      code: `import * as React from "react";
```
Comment (thatblindgeye) [inline packages/eslint-plugin-pf-codemods/test/rules/v5/pagination-rename-props.js:114]: Any reason for including the import from react?

Comment (adamviktora) [reply packages/eslint-plugin-pf-codemods/test/rules/v5/pagination-rename-props.js:114]: just copied it from the file I am testing the codemod on, I keep it there to not have ugly red errors, but it is unnecessary in the test, removing it now

**Comment by adamviktora** [comment]
seems like this codemod change will not resolve the https://github.com/patternfly/pf-codemods/issues/508 issue since the problem is related to using a wrapper component (`PaginationToolbarTemplate`)

**Review by thatblindgeye — APPROVED** [review]
While this may not resolve the linked issue, it wouldn't hurt to get in anyways just to expand the possible fixes.

**Review by gitdallas — APPROVED** [review]

**Pull request merged**